### ISSUE TITLE
adopt SynthMix Python script for FORCE

### ISSUE THREAD

**Issue by AndreasRabe** [davidfrantz/force#17]
testcase:
[synthmix.prm.txt](https://github.com/davidfrantz/force/files/4551181/synthmix.prm.txt)

**Comment by AndreasRabe** [comment]
in prm-file replace ITERATIONS with ENSEMBLES

**Comment by davidfrantz** [comment]
Thanks to @AndreasRabe , SynthMix is now available within FORCE :)

https://github.com/davidfrantz/force/commit/f21fd235a1853efbc4a432a3da5683c455fdf900

For details, see Akpona Okujeni's original paper: [Okujeni et al. "Support vector regression and synthetically mixed training data for quantifying urban land cover." Remote Sensing of Environment 137 (2013): 184-197](https://www.sciencedirect.com/science/article/pii/S0034425713002009)

SynthMix is heavily used in our lab to generate training data for machine learning regression to map sub-pixel fractions of land cover, tree cover etc. 

For a recent example, see [Schug et al. "Mapping urban-rural gradients of settlements and vegetation at national scale using Sentinel-2 spectral-temporal metrics and regression-based unmixing with synthetic training data." Remote Sensing of Environment 246 (2020): 111810](https://www.sciencedirect.com/science/article/pii/S0034425720301802)
